mindsdb/streams/kafka/kafka_stream.py

```diff
@@ -1,106 +1,24 @@
 import json
-from threading import Thread
-
 import kafka
 
-from mindsdb.utilities.log import log
-from mindsdb.utilities.cache import Cache
```

Comment:
Please use `Cache` for TS caching. It is based on `shelve` python package or Redis